Package.swift

```diff
@@ -19,7 +19,7 @@ let package = Package(
     // Dependencies declare other packages that this package depends on.
     // .package(url: /* package url */, from: "1.0.0"),
     .package(url: "https://github.com/google/swift-benchmark.git", .branch("master")),
-    .package(url: "https://github.com/saeta/penguin.git", .branch("master")),
+    .package(url: "https://github.com/saeta/penguin.git", .branch("marcrasi/arraybuffer_methods")),
```

Comment:
I will change this back to "master" once https://github.com/saeta/penguin/pull/70 gets merged into penguin.